src/dbm/DBMPoint.vb

```diff
@@ -39,6 +39,7 @@ Namespace Vitens.DynamicBandwidthMonitor
 
     Public DataManager As DBMDataManager
     Private Predictions As New Dictionary(Of DateTime, DBMPrediction)
+    Private PredictionsQueue As New Queue(Of DateTime) ' Maintains insertion order of the Predictions Dictionary

```

Comment:
Added queue in https://github.com/Vitens/DBM/commit/78b0e8d8b8ae0f82eb6fd48b669dbf2846201ac7